### ISSUE TITLE
Will be nice have example how to listen changes for parents

### ISSUE THREAD

**Issue by MaxGraey** [renatorib/react-powerplug#10]
So now I can see it like this for example:

```js
class Checkbox extends PureComponent {
  static propTypes = {
     onOutsideChange: PropTypes.func.isRequired,
  }

  render() {
    const { onOutsideChange, checked } = this.props;
    return (
      <Toggle initial={ checked }>
        {({ on, toggle }) => (
          <input
            type='checkbox'
            checked={ on }
            onChange={ () => { toggle(), onOutsideChange(!on) } }
          />
        )}
      </Toggle>
    );
  }
}
```

Can we do It in easier way? For example:

```js
    const { onOutsideChange, checked } = this.props;
    return (
      <Toggle initial={ checked } onToggle={ onOutsideChange }>
        {({ on, toggle }) => (
          <input
            type='checkbox'
            checked={ on }
            onChange={ toggle }
          />
        )}
      </Toggle>
```

**Comment by renatorib** [comment]
Oh yeah. It's a thing on my mind!

I was thinking these days about have a prop `onChange` on all components.
Now I am convinced to implement.

**Comment by renatorib** [comment]
Something like this:

```js
const callback = on => doSomething(on)

<Toggle onChange={callback}>{({ on, toggle }) => (
 ...
)}</Toggle>
```

**Comment by MaxGraey** [comment]
Yep, just the thing!

**Comment by renatorib** [comment]
Hey @MaxGraey 
I've published v0.1.2 with `onChange` callback on all components.

Still needs a documentation, but you can use it now.

```jsx
<Counter initial={3} onChange={({ count }) => { console.log(count) })}>
  {({ count, inc }) => (
    <button onClick={inc}>[{count}] Inc</buton>
  )}
</Counter>
```
